src/class_resolver/api.py

```diff
@@ -118,6 +118,16 @@ def get_option(self, *flags: str, default: Optional[str] = None, **kwargs):
             **kwargs,
         )
 
+    @property
+    def options(self) -> Collection[str]:
```

Comment:
Why not make it as specific as possible?

```suggestion
def options(self) -> Set[str]:
```